dev/deps/spark-deps-hadoop-2.2

```diff
@@ -116,7 +116,7 @@ libfb303-0.9.2.jar
 libthrift-0.9.2.jar
 log4j-1.2.17.jar
 lz4-1.3.0.jar
-mesos-0.21.1-shaded-protobuf.jar
+mesos-0.28.2-shaded-protobuf.jar
```

Comment:
Are there any other known implications to bumping through 7 minor releases of Mesos? that seems worth vetting